audio/descriptors.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2017 Kyle Robbertze
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
```

Comment:
If you are talking about general GPLv2/GPLv2 stuff - this info is taken directly from both the Wikipedia page on license compatibility and the FSF page on the same thing.

If you are talking about the fx2lib stuff specifically, then welp, what you waiting for? I know you know how to send pull requests and there are wikis at https://github.com/mithro/fx2lib/wiki & https://github.com/timvideos/HDMI2USB/wiki if you want to put things there.